odoo/tools/misc.py

```diff
@@ -1380,6 +1380,26 @@ def _format_time_ago(env, time_delta, lang_code=False, add_direction=True):
     return babel.dates.format_timedelta(-time_delta, add_direction=add_direction, locale=locale)
 
 
+def format_number(number, decimal=1):
+    for unit in ['', 'k', 'M', 'G']:
+        if abs(number) < 1000.0:
+            return "%g%s" % (round(number, decimal), unit)
+        number /= 1000.0
+    return "%g%s" % (round(number, decimal), 'T')
```

Comment:
Could you add a docstring for both methods ? Indeed you should explain choices you made about rounding, supporting only k / M / G (why not more ?), and give some examples. Do the same for the other formatting method, some explanation would be welcomed.